### DISCUSSION TITLE
Hugging Face - Private repo

### DISCUSSION THREAD

**Discussion by NamelessButler** [etherealxx/batchlinks-webui#24] [Ideas]
Is it possible to download from huggingface private repos using tokens ?

**Comment by etherealxx** [comment]
@falsonerd haven't thought about that honestly. I might add that feature if it could be useful for you. Maybe using `@token <token_code>` or something.

**Reply by NamelessButler** [reply]
It would be appreciated.
I usually use HuggingFace api to download  from private repo.
It also let's me download very fast considering private repo speed limitation.
And @token would be very convenient...
